GooglePlayInstant/Samples/SphereBlast/scripts/ThirdPersonCamera.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class thirdPersonCamera : MonoBehaviour {
+
+    Quaternion rotation;
+    public Transform sphere;
+    Vector3 offset;
+
+    void Start () {
+        offset = transform.position - sphere.transform.position;
+    }
+
+    void FixedUpdate()
+    {
+        if (sphere == null)
```

Comment:
Where is "sphere" set?

Reply:
Sphere is set in the Unity scene. It's a quirk of Unity development to have some values set through the scene, but common in the code I've looked at.